### PR TITLE
az, az-apim, az-group, az-storage, az-tag, az-upgrade: add Spanish translation

### PR DESCRIPTION
**Pull request by crixodia**
Added Spanish translation for the following pages:

- az-apim
- az,
- az-storage
- az-group
- az-tag
- az-upgrade

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution.

Note to maintainers: This PR can be reviewed for files other than az-storage, we can review it after #10947's merge.

**Comment by kant** [comment]
@kbdharun ok

File: pages.es/common/az-apim.md
Comment (kant) [inline pages.es/common/az-apim.md:11]: Crear --> Crea

File: pages.es/common/az-apim.md
Comment (kant) [inline pages.es/common/az-apim.md:19]: instancia de servicio --> instancia del servicio

**Review by kant — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by kant — COMMENTED** [review]
LGTM

**Pull request merged**
